specs/merge/beacon-chain.md

```diff
@@ -214,6 +216,36 @@ def is_execution_enabled(state: BeaconState, body: BeaconBlockBody) -> bool:
     return is_merge_block(state, body) or is_merge_complete(state)
 ```
 
+### Genesis block
+
+#### `get_genesis_block_from_genesis_state`
```

Comment:
I think for Merge specs, it would be better to move this function to under the "Testing" section as it's testing (testnet) only.

Reply:
I think it's okay to keep it here as is. (Similar to `initialize_beacon_state_from_eth1`)